### PR TITLE
Update Readme before gem creation

### LINKED ISSUE #4: README

### PR DESCRIPTION
**Pull request by DerekStride**
closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,22 +22,107 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+Add an instance of GraphQL::Analyzer to your schema, instantiate it with a list of instrumentation to capture different datastore queries.
```
Comment (justinkrol) [inline README.md]: list of instrumentation**s**

Comment (justinkrol) [reply README.md]: Maybe would be helpful to explicitly list the instrumentations that are already implemented?

File: README.md [hunk 2]
```diff
@@ -22,22 +22,107 @@ Or install it yourself as:
...
+              ### OMITTED ###
+          end
+        end
+      end
+    end
+  end
+end
+```
+
+### Response Format
```
Comment (justinkrol) [inline README.md]: Given that the instrumentation section is optional, what do you think of putting it below the Response Format section?

**Review by justinkrol — APPROVED** [review]
Minor things, but I'm good with it as is

File: README.md (hunk 2 above)
Comment (DerekStride) [reply README.md]: 👍 I'll also add the list of already implemented instruments to this section.

**Pull request merged**